test/common/config/BUILD

```diff
@@ -94,28 +94,6 @@ envoy_cc_test(
     ],
 )
 
-envoy_cc_test(
```

Comment:
Did we lose coverage here somehow?

Reply:
this class contained a single test which seemed to be duplicated in grpc_mux_impl_test.cc

Reply:
Specifically, TEST_F(GrpcMuxImplTest, DiscoveryResponseNonexistentSub) which does everything this test does and adds a few additional checks.